python/PiFinder/setup.py

```diff
@@ -163,8 +164,67 @@ def insert_catalog_max_sequence(catalog_name):
     conn.commit()
 
 
+def resolve_object_images():
+    # This is the list of catalogs to search for
+    # objects to match against image names
+    resolution_priority = [
```

Comment:
nothing wrong with this for now, but would like to have an extra column in the catalog table with the priority for that catalog

Reply:
I've changed this to use the insert order of the catalogs for priority.  I slightly adjusted the catalog import order and added a comment to remind future folks that the order is important